contributing.md

```diff
@@ -1,3 +1,27 @@
+Contributing to the Ssh Linux System Role
+=============================================
+
+Where to start
+--------------
+
+The first place to go is [Contribute](https://linux-system-roles.github.io/contribute.html).
+This has all of the common information that all role developers need:
+* Role structure and layout
+* Development tools - How to run tests and checks
+* Ansible recommended practices
+* Basic git and github information
+* How to create git commits and submit pull requests
+
+- **Bugs and needed implementations** are listed on [Github
+  Issues](https://github.com/linux-system-roles/ssh/issues). Issues labeled with
+[**help
+wanted**](https://github.com/linux-system-roles/ssh/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
+are likely to be suitable for new contributors!
+
+- **Code** is managed on
+  [Github](https://github.com/linux-system-roles/ssh), using [Pull
+Requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests).
+
 # ssh
```

Comment:
It would be good to sync also the rest of the file with the semantics of the standard contributing.md:
 * Use `---` and `===` for headers instead of `#`
 * Provide some more specific description here instead of `ssh`. For example "Role specific notes", or something?